### DISCUSSION TITLE
What is the plan for migrating to theme-ui@0.6 and emotion@11?

### DISCUSSION THREAD

**Discussion by RishikeshDarandale** [reflexjs/reflexjs#155] [Q&A]
`theme-ui`@`0.6` is released 2 days ago with [Change log](https://github.com/system-ui/theme-ui/blob/develop/CHANGELOG.md). What's the plan of `reflexjs` to support the latest version?

**Comment by shadcn** [comment]
I've started looking into it. A few breaking changes but so far so good. We'll also add support for the new JSX runtime and React 17. I'll keep this thread updated.

Thank you

**Reply by RishikeshDarandale** [reply]
Thanks @arshad !

**Comment by shadcn** [comment]
Update: Made some progress on this. Things are looking good.

## Notes

1. We are planning to make this a major release with bug fixes and new features.
2. There are some breaking changes but should be easy to upgrade.

I've updated the Reflexjs site to use `reflexjs 2.0.0-alpha2`, `react 17.0.2` (with new automatic JSX runtime) and `theme-ui 0.6.1`.

Everything looking good so far: https://next.reflexjs.org

**Comment by shadcn** [comment]
We released Reflexjs 2.0.0 yesterday. You can find the upgrade guide here: https://reflexjs.org/docs/upgrade

Thank you

**Reply by RishikeshDarandale** [reply]
@arshad , How are you planning to get the lastest from `theme-ui`? I see now `0.7.3` is released!

**Reply by shadcn** [reply]
Probably next week. I ran the update locally and everything looking good. I'll tag a new release next week.
